spec/sassdocs_helpers_spec.rb

```diff
@@ -0,0 +1,319 @@
+require 'json'
+require 'ostruct'
+require 'active_support/all'
+
+require_relative '../sassdocs_helpers.rb'
+
+# Middleman helpers convert data to allow dot access, so we need to bring in data in a similar way.
+def dothash(hash)
+  JSON.parse(hash.to_json, object_class: OpenStruct)
+end
+
+RSpec.describe SassdocsHelpers do
+  before(:each) do
+    allow(File).to receive(:read).and_return('{ "dependencies": { "govuk-frontend": { "version": "1.0.0" } } }')
+    # Include mixin into a test class to allow us to mock File
+    class Test
+      include SassdocsHelpers
+    end
+    @helper = Test.new
+  end
+
+  describe '#format_sassdoc_data' do
+    it "should raise an error if there is no data" do
```

Comment:
I don't know if we follow this elsewhere, but my personal preference is not to use should in the descriptions.

http://www.betterspecs.org/#should